iron-fish/ironfish
# stores identity in multisig account value

## Summary

updates the MultisigKeys interface to store the multisig account identity in the account value

adds a migration, 033, that derives the account identity from the account secret already stored in the account

updates importAccount to derive identity from secret at time of import. deriving the identity instead of using the identity included in the import maintains backwards compatibility

Closes IFL-3118

## Testing Plan
1. export multisig account as JSON on staging, see that it does not contain identity
2. export multisig account as JSON on this branch, see that it does contain identity
3. import exports back into new datadirs on this branch, verify that imports succeed

## Documentation

Does this change require any updates to the Iron Fish Docs (ex. [the RPC API
Reference](https://ironfish.network/docs/onboarding/rpc/chain))? If yes, link a
related documentation pull request for the website.

```
[ ] Yes
```

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and label it with `breaking-change-rpc` or `breaking-change-sdk`.

```
[ ] Yes
```
